models/action.go

```diff
@@ -674,3 +674,17 @@ func GetFeeds(ctxUser *User, actorID, offset int64, isProfile bool) ([]*Action,
 	err := sess.Find(&actions)
 	return actions, err
 }
+
+// ChangeUsernameInAction changes the name of act_user_name & repo_user_name
+func ChangeUsernameInAction(id int64, newUserName string) error {
+	ac := Action{
+		RepoUserName: newUserName,
+		ActUserName:  newUserName,
+	}
+	_, err := x.
+		Cols("act_user_name", "repo_user_name").
+		Where("act_user_id = ?", id).
+		Update(ac)
+	log.Info("%v", err)
```

Comment:
I agree, I use one from [pull.go#1030-1039](https://github.com/go-gitea/gitea/blob/master/models/pull.go#L1038) and there is no test there.
Anyway, I will make a test for the nil value.